### PR TITLE
feat: update version of near-api-js

### PR DESCRIPTION
**Pull request by fospring**
dependency of near-api-js was too old，update it from `0.44.1`  to `2.1.4`

### CODE REVIEW

File: packages/js/src/account/account-manager.ts [hunk 1]
```diff
@@ -217,7 +216,7 @@ export abstract class AccountManager implements NearAccountManager {
   }
 
   protected get connection(): nearAPI.Connection {
-    return new nearAPI.Connection(this.networkId, this.provider, this.signer);
+    return new nearAPI.Connection(this.networkId, this.provider, this.signer, `jsvm.${this.networkId}`);
```
Comment (ailisp) [inline packages/js/src/account/account-manager.ts:219]: Is this added by mistake?

File: packages/js/dist/transaction-result.d.ts
Comment (ailisp) [inline packages/js/dist/transaction-result.d.ts:1]: transaction-result.ts is not changed, is dist rebuilt after resolve the conflict?

Comment (fospring) [reply packages/js/dist/transaction-result.d.ts:1]: yep, I rebuit it in the directory of `packages/js`

File: packages/js/src/account/account-manager.ts (hunk 1 above)
Comment (fospring) [reply packages/js/src/account/account-manager.ts:219]: No, constructor of `Connection` changed: https://github.com/near/near-api-js/blob/master/packages/accounts/src/connection.ts#L41

**Review by ailisp — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**